### DISCUSSION TITLE
J'essaie simplement

### DISCUSSION THREAD

**Discussion by Toumic** [Quefumas/gensound#39] [Q&A] [answered]
Bonjour.

J'ai créé une application qui produit des gammes musicales.
Jusque-là, tout va bien. Mais quand je veux écouter les gammes, ça ne fonctionne pas.

```
# Python UTF8

from gensound import Sine, Gain

# Liste des fréquences
hertz=[2093.004522404789, 2349.3181433392606, 329.6275569128699, 2793.825851464031,
       6271.926975707988, 116.54094037952248, 246.94165062806206]
for freq in hertz: 
    dure   2.0  # Durée en secondes
    onde = Sine(frequency=freq, duration=duree) * Gain(-6)  # Atténuation de 6 dB

    # Joue l'onde
    onde.play()
```

Je n'ai pas de message d'erreur et je n'entends rien. 
Je serais content de lire des réponses simples, car mon niveau est faible dans ce domaine.

**Comment by Toumic** [comment, marked as answer]
En fait, Gensound crée bien un fichier son.wav. Ce son est placé dans un dossier gensound.temp.

N'hésitez pas à parler de théorie musicale avec moi. ♥☺♥

**Comment by Quefumas** [comment]
Hello Toumic,

I'm happy you are enjoying the code!
The sound.wav file is only one way to play the sound. Try running this:
```python
from gensound.io import IO
IO.status()
```
This will let you know if you have other ways to play sound directly. Maybe you will need to install `simpleaudio` or `pygame`.

I hope it works!
